plugins/processors/batch/sample.conf

```diff
@@ -0,0 +1,14 @@
+## Batch metrics into separate batches by adding a tag indicating the batch index.
+## Can be used to route batches of metrics to different outputs
+## to parallelize writing of metrics to an output
+## Metrics are distributed across batches using the round-robin scheme.
```

Comment:
Please move this to the README. By convention the header above the plugins should be only one line...